### DISCUSSION TITLE
Anybody has successfully run k8s on Hetzner GPU nodes (GEX44)?

### DISCUSSION THREAD

**Discussion by seperman** [vitobotta/hetzner-k3s#371] [Q&A]
Hello,
Anybody has successfully run k8s on Hetzner GPU nodes (GEX44) using hetzner-k3s?
If yes, did the GPU sharing work fine? Was the overhead of k8s minimal for GPU?
Thanks

**Comment by vitobotta** [comment]
I am curious about these GPUs. How good are they for machine learning stuff?

**Reply by jrudolph** [reply]
Wikipedia has good tables to compare specs. I think the main point is the 4000 SFF's low power consumption at only 70W. Just looking at TFlops a 4090 might be 4-8x faster at 6-7x power consumption.
So it might be used more for inference than for training I guess.

https://en.m.wikipedia.org/wiki/Quadro
https://en.m.wikipedia.org/wiki/GeForce_40_series
